<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DictateWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTreeWidget" name="resource_list">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>561</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="progress_slider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>541</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="start_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="pause_button">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="progress_time_lable">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>00:00/00:00</string>
       </property>
      </widget>
      <widget class="QSlider" name="volume_slider">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>60</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>volume</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="script_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>601</width>
        <height>291</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_button">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
